staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1059,7 +1060,7 @@ func (e *Store) Delete(ctx context.Context, name string, deleteValidation rest.V
 	}
 
 	// delete immediately, or no graceful deletion supported
-	klog.V(6).Infof("going to delete %s from registry: ", name)
+	klog.V(6).InfoS("going delete obj from registry", "obj", name)
```

Comment:
s/obj/object,

s/going/Going

and can you use `klog.KObj(obj)` instead of name?

Reply:
I can't get the obj by the name in the context of this function, so I keep using the name.